### PR TITLE
LLM: add internlm example on arc

### PR DESCRIPTION
**Pull request by plusbang**
## Description

Add Linux arc internlm example.

### 4. How to test?
- [x] Local test

### CODE REVIEW

File: python/llm/example/transformers/transformers_int4/GPU/internlm/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+In the example [generate.py](./generate.py), we show a basic use case for a InternLM model to predict the next N tokens using `generate()` API, with BigDL-LLM INT4 optimizations on Intel® Arc™ A-Series Graphics.
+### 1. Install
+We suggest using conda to manage environment:
+```bash
+conda create -n llm python=3.9
+conda activate llm
+# below command will install intel_extension_for_pytorch==2.0.110+xpu as default
+# you can install specific ipex/torch version for your need
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+pip install --pre --upgrade bigdl-core-xe
```
Comment (rnwang04) [inline python/llm/example/transformers/transformers_int4/GPU/internlm/README.md:17]: This line can be removed now.

File: python/llm/example/transformers/transformers_int4/GPU/llama2/README.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ conda create -n llm python=3.9
 conda activate llm
 # below command will install intel_extension_for_pytorch==2.0.110+xpu as default
 # you can install specific ipex/torch version for your need
-pip install bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
-pip install bigdl-core-xe
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+pip install --pre --upgrade bigdl-core-xe
```
Comment (rnwang04) [inline python/llm/example/transformers/transformers_int4/GPU/llama2/README.md:17]: the same for before examples.

**Review by rnwang04 — APPROVED** [review]
others LGTM

File: python/llm/example/transformers/transformers_int4/GPU/internlm/README.md (hunk 1 above)
Comment (plusbang) [reply python/llm/example/transformers/transformers_int4/GPU/internlm/README.md]: > This line can be removed now.

okay, remove it in all examples.

**Pull request merged**